src/java/main/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -621,7 +621,7 @@ public Listener() {
         @Override
         public void run() {
             int numRetries = 0;
-            InetSocketAddress addr;
+            InetSocketAddress addr = null;
```

Comment:
why is this needed?

Reply:
for logging the **addr** in LINE_657

Reply:
@maoling I think a test case would be great because catching that exception does "change behavior".